src/bin/plugins/checkPlugin.js

```diff
@@ -429,7 +429,7 @@ fs.readdir(pluginPath, (err, rootFiles) => {
   // The ep_etherpad-lite peer dep must be installed last otherwise `npm install` will nuke it. An
   // absolute path to etherpad-lite/src is used here so that pluginPath can be a symlink.
   execSync(
-      `${npmInstall} --no-save ep_etherpad-lite@file:${__dirname}/../../src`, {stdio: 'inherit'});
+      `${npmInstall} --no-save ep_etherpad-lite@file:${__dirname}/../../`, {stdio: 'inherit'});
```

Comment:
Shouldn't this still be `../../src`?

Reply:
``src`` broke when I ran it, changed it to the ``../../`` and it worked...  Didn't go further into than that!

Reply:
Oops, I misread the code. `__dirname` is the directory containing the `checkPlugin.js` script, not the plugin's directory. Your change is correct.